<?xml version="1.0" encoding="utf-8"?>
<TestRun id="2f215988-022c-4fa6-9adb-302091ceb538" name="anna@cloudtest-win10 2023-01-22 23:34:04" runUser="cloudtest-win10\anna" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <Times creation="2023-01-22T23:34:04.1709055+00:00" queuing="2023-01-22T23:34:04.1709060+00:00" start="2023-01-22T23:33:57.5964877+00:00" finish="2023-01-22T23:34:04.2072796+00:00" />
  <TestSettings name="default" id="b23102b2-37f8-43a1-b351-125a7a29c024">
    <Deployment runDeploymentRoot="anna_cloudtest-win10_2023-01-22_23_34_04" />
  </TestSettings>
  <Results>
    <UnitTestResult executionId="3f09f4a3-654b-4215-ad46-91441ec3a35d" testId="60a46c89-261b-3ab8-4a85-671ac4a81765" testName="Tester.AzureUtils.AzureTableDataManagerTests.AzureTableDataManager_DeleteTableAsync" computerName="cloudtest-win10" duration="00:00:01.7420929" startTime="2023-01-22T23:34:03.8646103+00:00" endTime="2023-01-22T23:34:03.8646584+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="3f09f4a3-654b-4215-ad46-91441ec3a35d" />
  </Results>
  <TestDefinitions>
    <UnitTest name="Tester.AzureUtils.AzureTableDataManagerTests.AzureTableDataManager_DeleteTableAsync" storage="c:\users\anna\orleans\test\extensions\testerazureutils\bin\debug\net7.0\tester.azureutils.dll" id="60a46c89-261b-3ab8-4a85-671ac4a81765">
      <Execution id="3f09f4a3-654b-4215-ad46-91441ec3a35d" />
      <TestMethod codeBase="C:\Users\anna\orleans\test\Extensions\TesterAzureUtils\bin\Debug\net7.0\Tester.AzureUtils.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="Tester.AzureUtils.AzureTableDataManagerTests" name="AzureTableDataManager_DeleteTableAsync" />
    </UnitTest>
  </TestDefinitions>
  <TestEntries>
    <TestEntry testId="60a46c89-261b-3ab8-4a85-671ac4a81765" executionId="3f09f4a3-654b-4215-ad46-91441ec3a35d" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <ResultSummary outcome="Completed">
    <Counters total="1" executed="1" passed="1" failed="0" error="0" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" completed="0" inProgress="0" pending="0" />
    <Output>
      <StdOut>[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.4.5+1caef2f33e (64-bit .NET 7.0.2)&#xD;
[xUnit.net 00:00:01.27]   Discovering: Tester.AzureUtils (method display = ClassAndMethod, method display options = None)&#xD;
[xUnit.net 00:00:01.49]   Discovered:  Tester.AzureUtils (found 246 test cases)&#xD;
[xUnit.net 00:00:01.50]   Starting:    Tester.AzureUtils (parallel test collections = off, max threads = 4)&#xD;
[xUnit.net 00:00:03.35]   Finished:    Tester.AzureUtils&#xD;
</StdOut>
    </Output>
    <RunInfos>
      <RunInfo computerName="cloudtest-win10" outcome="Warning" timestamp="2023-01-22T23:34:04.0222865+00:00">
        <Text>Data collector 'Blame' message: All tests finished running, Sequence file will not be generated.</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
</TestRun>